backend/pick-git/src/main/java/com/woowacourse/pickgit/user/exception/DuplicatedFollowException.java

```diff
@@ -0,0 +1,10 @@
+package com.woowacourse.pickgit.user.exception;
+
+public class DuplicatedFollowException extends PickgitException {
+
+    private static final String MESSAGE = "이미 팔로우 중 입니다.";
```

Comment:
```suggestion
    private static final String ERROR_CODE = "U0001";
```

에러 메시지 대신 에러 코드로 변경해주세요!